lib/components/src/blocks/PropsTable/PropsTable.stories.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { PropsTable, PropsTableError } from './PropsTable';
+import { DocsPageWrapper } from '../DocsPage';
+import * as rowStories from './PropRow.stories';
+
+export default {
+  Component: PropsTable,
+  title: 'Docs|PropTable',
+  decorators: [storyFn => <DocsPageWrapper>{storyFn()}</DocsPageWrapper>],
+};
+
+export const error = () => <PropsTable error={PropsTableError.NO_COMPONENT} />;
+
+export const empty = () => <PropsTable rows={[]} />;
+
+const { row: stringRow } = rowStories.string().props;
+const { row: numberRow } = rowStories.number().props;
```

Comment:
:+1: how do you like this style @shilman

Reply:
it works. i think exporting data will be cleaner tho!